private/buf/cmd/buf/command/mod/modupdate/modupdate.go

```diff
@@ -113,6 +113,20 @@ func run(
 	remote := bufrpc.DefaultRemote
 	if moduleConfig.ModuleIdentity != nil && moduleConfig.ModuleIdentity.Remote() != "" {
 		remote = moduleConfig.ModuleIdentity.Remote()
+	} else {
+		for _, moduleReference := range moduleConfig.Build.DependencyModuleReferences {
+			if moduleReference.Remote() != bufrpc.DefaultRemote {
+				warnMsg := fmt.Sprintf(
```

Comment:
Perhaps we should restrict this warning to `buf.teams` for now? Otherwise this'll also show if people add a `github.com/...` dep and suggest to add github to `name:`.

Reply:
Yeah, I think that might makes sense... I can see how this could be confusing to a new user of `buf` thinking that we can support VCS remotes...